fdbserver/ptxn/test/TestResolver.actor.cpp

```diff
@@ -109,5 +139,24 @@ TEST_CASE("fdbserver/ptxn/test/resolver") {
 		ASSERT(f.isReady() && f.isValid());
 	}
 
+	// Verify team's PCVs are correct.
```

Comment:
I had to think for a second before realizing PCV is previous commit version. Maybe from perspective of readers who don't actively work on this, calling it "previous CV" or even CommitVersion is better, as it doesn't break rhythm of new readers.